README.md

```diff
@@ -1,11 +1,11 @@
-# narendrach-research-reflection
+# Narendra's Research and Reflection Journal 
```

Comment:
I made some formatting changes throughout to demonstrate how you can better organize your README.md for readability. Please make sure to use proper headings (i.e. the `#`, `##`, etc.) since these provide more accurate semantic information.